chromium/README.md

```diff
@@ -1,4 +1,8 @@
-This directory contains instructions and corresponding patches to build the Chromium web browser using the [OQS-BoringSSL fork](https://github.com/open-quantum-safe/boringssl), thereby enabling Chromium to use quantum-safe key exchange algorithms. Note that these instructions have been tested only on Windows 10, Ubuntu 18, 19, and 20 (x86_64) installations and apply at present only to a subset of quantum-safe key-exchanges as [documented here](https://github.com/open-quantum-safe/boringssl#key-exchange).
+This directory contains no longer fully maintained instructions and corresponding patches to build the Chromium web browser using the [OQS-BoringSSL fork](https://github.com/open-quantum-safe/boringssl), thereby enabling Chromium to use quantum-safe key exchange algorithms.
+
+These instructions are based on liboqs `0.8.0` and Chromium `117.0.5863.0`; they have been tested only on Windows 10 and Ubuntu 22 LTS(x64) installations and apply at present only to a subset of quantum-safe key-exchanges as [documented here](https://github.com/open-quantum-safe/boringssl#key-exchange).
```

Comment:
It may be prudent to add explicit wording stating that the instructions and code (patchfile) _must_ use these liboqs and Chromium code versions. Anything else _will_ fail. And/or add wording welcoming contributions to pull this integration (obviously now limited to the patchfile) to more current up- and downstream code versions? Maybe something like "These instructions are known to work only on ..... We welcome any contributions to update the patchfile to more current versions of liboqs and Chromium." Maybe the latter sentence only at the very end of the section below (together with the re-wording suggested there?